src/Perms/perm_images.jl

```diff
@@ -92,36 +92,36 @@ else
     function Base.copy(p::Perm)
         imgs = copy(p.images)
         q = typeof(p)(imgs; check = false)
-        if isdefined(p, :inv, :sequentially_consistent)
-            inv_imgs = copy(@atomic(p.inv).images)
+        if isdefined(p, :inv)
+            inv_imgs = copy(p.inv.images)
             q⁻¹ = typeof(p)(inv_imgs; check = false)
-            @atomic q.inv = q⁻¹
-            @atomic q⁻¹.inv = q
+            q.inv = q⁻¹
+            q⁻¹.inv = q
```

Comment:
You did have a bad mistake here (of generally the error sort, though generally not the crash sort), since you assigned these in the wrong order, the `:sequentially_consistent` annotation is invalidated everywhere and invalid.
```suggestion
            q⁻¹.inv = q
            q.inv = q⁻¹
```